### PR TITLE
Watch branding factory colors directly

### LINKED ISSUE #1249: [Branding Component] Preview lag
**Defect:** There's a lag between making a color change and it being reflected in preview. See this video for context:

https://www.loom.com/share/872812df0bbc402db7d66db4ff259cff

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Preview holder watches colors directly linked to text boxes
so that preview updates on every color change

Remove watch for company.updated message

Only perform logo check on show function for image component

[stage-18]

## Motivation and Context
Fix #1249 

## How Has This Been Tested?
This was tested in the local environment using the preview holder workaround for a *.risevision.com domain.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/scripts/template-editor/components/directives/dtv-component-image.js [hunk 1]
```diff
@@ -86,6 +86,12 @@ angular.module('risevision.template-editor.directives')
           function _loadSelectedImages() {
             var selectedImages = imageFactory.getImagesAsMetadata();
 
+            // Show logo in the image list if component is set to use logo and a logo is available
```
Comment (ezequielc) [inline web/scripts/template-editor/components/directives/dtv-component-image.js]: @alex-deaconu By moving this to here, an issue was introduced. If a component is using branding logo and I pick a new file for that component, two image files are listed. Thanks

Comment (alex-deaconu) [reply web/scripts/template-editor/components/directives/dtv-component-image.js]: @ezequielc good catch, will revert. Thanks!

Comment (alex-deaconu) [reply web/scripts/template-editor/components/directives/dtv-component-image.js]: Updated, thanks!

**Pull request merged**